db/migrate/20230117150113_create_likes.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes, &:timestamps
+  end
+end
```

Comment:
- your table should contain references this will create author_id and a corresponding index 😄
- your table should contain references this will create author_id and a corresponding index (but students can also add column and index in separate commands) 😄 👍🏾